pkg/operator/controller/garden/garden/components.go

```diff
@@ -1163,6 +1168,10 @@ func (r *Reconciler) newPrometheusLongTerm(log logr.Logger, garden *operatorv1al
 			SigningCA:              operatorv1alpha1.SecretNameCARuntime,
 			WildcardCertSecretName: wildcardCertSecretName,
 		},
+		Cortex: &prometheus.CortexValues{
+			Image:         imageCortex.String(),
+			CacheValidity: 168 * time.Hour,
```

Comment:
Do we want to add a comment indicating the time frame in a more readable fashion or split it into `7 * 24` so that it is more obvious that this is a week?